sql/item_strfunc.cc

```diff
@@ -1449,6 +1449,42 @@ String *Item_func_sformat::val_str(String *res)
   return null_value ? NULL : res;
 }
 
+
+#include <openssl/rand.h>
+
+bool Item_func_random_bytes::fix_length_and_dec() { return FALSE; }
```

Comment:
minor nit, very small functions like this can be in the include header.

Reply:
why was it needed?

Reply:
1. add `create table t1 as select random_bytes(100)` to your test.
2. try
```sql
select seq from seq_0_to_255 where seq = ord(random_bytes(1))
```
don't add it to the test, it's non-deterministic. but it should return different number of rows on different runs. try with `rand()` for comparison.